themes/dark/templates/addShows_addExistingShow.mako

```diff
@@ -104,8 +113,23 @@ const startVue = () => {
                     return;
                 }
 
-                const params = { 'rootDirs': indices.join(',') };
-                const { data } = await api.get('internal/existingSeries', { params });
+                let data = null;
+                const config = {
+                    params: {
+                        'rootDirs': indices.join(',')
+                    },
+                    timeout: this.requestTimeout
+                };
+                try {
+                    const response = await api.get('internal/existingSeries', config)
```

Comment:
If you need a try/catch please use `.then` and `.catch`

```js
api.get('internal/existingSeries', config).then(response => {
    data = response.data;
}).catch(error => {
    this.errorMessage = error.message;
    this.isLoading = false;
    this.dirList = [];
    return;
});
```

Reply:
Also, I know have to put everything inside the `then` block, or move it to a new function, because the function continues to run without waiting for the response...